docs/local_install_skaffold.md

```diff
@@ -0,0 +1,190 @@
+# Local install of Substra using kubernetes and skaffold
+
+This guide will help you run the Substra platform on your machine with a 2 organisations setup.
+
+- [Setup](#setup)
+  - [1. Get the source code](#1-get-the-source-code)
+  - [2. Install Docker and Kubernetes](#2-install-docker-and-kubernetes)
+  - [3. Configure Kubernetes](#3-configure-kubernetes)
+  - [4. Install skaffold](#4-install-skaffold)
+- [Running the platform](#running-the-platform)
+- [Login, password and urls](#login-password-and-urls)
+
+## Setup
+
+### 1. Get the source code
+
+You'll need either download or clone 3 repositories:
+
+#### Download
+
+Download links:
+
+- [hlf-k8s](https://github.com/SubstraFoundation/hlf-k8s/archive/dev.zip)
+- [substra-backend](https://github.com/SubstraFoundation/substra-backend/archive/dev.zip)
+- [substra-frontend](https://github.com/SubstraFoundation/substra-frontend/archive/dev.zip)
+
+#### Git clone
+
+Git clone commands:
+
+- hlf-k8s: `git clone git@github.com:SubstraFoundation/hlf-k8s.git`
+- substra-backend: `git@github.com:SubstraFoundation/substra-backend.git`
+- substra-frontend: `git@github.com:SubstraFoundation/substra-frontend.git`
+
+### 2. Install Docker and Kubernetes
+
+First of all, download the *Docker desktop* installer from https://www.docker.com/products/docker-desktop. You'll have to create an account there to do so. Then run it to install Docker on your machine.
+
+Once installed, launch Docker and open its "preferences" panel. In the Kubernetes tab, check the `Enable Kubernetes` checkbox.
+
+Kubernetes will take a while to launch the first time, but once it is done, you can move on to configuring it as follow:
+
+### 3. Configure Kubernetes
+
+Using the command line, run first this command:
+
+```sh
+helm init
+```
+
+Once done, we can configure your machine so that the urls `substra-backend.node-1.com`, `substra-backend.node-2.com`, `substra-frontend.node-1.com` and `substra-frontend.node-2.com` point to your local instance of the platform.
+
+To do so, you'll first need to get the host IP address from container by running:
+
+```sh
+docker run -it --rm busybox ping host.docker.internal
+```
+
+This will output should look like this:
```

Comment:
Wording issue? `The output should look like this:`?

Reply:
👍